exporter/trace/trace_proto.go

```diff
@@ -178,11 +180,36 @@ func protoFromReadOnlySpan(s sdktrace.ReadOnlySpan, defaultTraceAttributes []att
 		sp.TimeEvents.DroppedAnnotationsCount = clip32(droppedAnnotationsCount)
 	}
 
-	// TODO(ymotongpoo): add implementations for Links
+	sp.Links = linksProtoFromLinks(s.Links())
 
 	return sp
 }
 
+func linksProtoFromLinks(links []trace.Link) *tracepb.Span_Links {
+	var linksPb *tracepb.Span_Links
```

Comment:
nit: you can instantiate a return value in the function definition, and avoid the extra declaration (and you don't need to specify anything when you return):
```golang
func linksProtoFromLinks(links []trace.Link) (linksPb *tracepb.Span_Links) {
    ...
    return
}
```

Reply:
(Sorry for the fly-by)

Please don’t use naked returns for this purpose. 

https://github.com/golang/go/wiki/CodeReviewComments#named-result-parameters

Reply:
I restructured this a bit